test/utils/expect.ts

```diff
@@ -1,47 +1,23 @@
 import { BigNumber, ethers } from 'ethers';
+import { BlockList } from 'net';
 import PoolData from '../types/PoolData';
 import UserData from '../types/UserData';
 import { calculateDataAfterUpdate, calculateRewardIndex, calculateUserReward } from './calculate';
 
 export function expectDataAfterStake(
-  poolData: PoolData,
-  userData: UserData,
-  txTimeStamp: BigNumber,
-  amount: BigNumber
-): [PoolData, UserData] {
-  const [newPoolData, newUserData]: [PoolData, UserData] = calculateDataAfterUpdate(
-    poolData,
-    userData,
-    txTimeStamp
-  );
-
-  const newUserStakingAssetBalance = newUserData.stakingAssetBalance.sub(amount);
-  const newUserPrincipal = newUserData.userPrincipal.add(amount);
-
-  newUserData.stakingAssetBalance = newUserStakingAssetBalance;
-  newUserData.userPrincipal = newUserPrincipal;
-
-  const newPoolTotalPrincipal = newPoolData.totalPrincipal.add(amount);
-  newPoolData.totalPrincipal = newPoolTotalPrincipal;
-
-  const newPoolStakingAssetBalance = newPoolData.stakingAssetBalance.add(amount);
-  newPoolData.stakingAssetBalance = newPoolStakingAssetBalance;
-
-  return [newPoolData, newUserData];
-}
-
-export function expectDataAfterStakeSetReward(
   poolData: PoolData,
   userData: UserData,
   txTimeStamp: BigNumber,
   amount: BigNumber,
   rewardPerSecond: BigNumber,
```

Comment:
I'm curious whether rewardPerSecond and duration in function parameter are different with these two in poolData.